hphp/runtime/vm/jit/vasm-lower.cpp

```diff
@@ -75,7 +75,9 @@ void lower_vcall(Vunit& unit, Inst& inst, Vlabel b, size_t i) {
       // there are an odd number of arguments.
       int numArgs = vargs.stkArgs.size();
       if (adjust) {
-        v << push{vargs.stkArgs[numArgs-1]};
+	// Using InvalidReg below fails SSA checks and simplify pass.
+        //v << pushp{vargs.stkArgs[numArgs-1], InvalidReg};
+        v << pushp{vargs.stkArgs[numArgs-1], vargs.stkArgs[numArgs-1]};
```

Comment:
This is where things really started to tank. The commented line is exactly what I wanted to do when pushing the stack arguments. However, this happens early enough that check SSA started to complain of the `InvalidReg`. So for this one place, I went ahead and just pushed that odd stack argument twice.

Depending on what you think, this could easily be a simple policy for every time we have to push an odd number of things. The only down side, is that a pushp{a, a} will lower (X64 and PPC64) to push{a} and push{a} which I imagine is slower than a `push` and `lea` which we could easily do if we supported `InvalidReg`.